src/libAtomVM/externalterm.c

```diff
@@ -131,6 +131,7 @@ term externalterm_to_term(const void *external_term, size_t size, Context *ctx,
 
 enum ExternalTermResult externalterm_from_binary(Context *ctx, term *dst, term binary, size_t *bytes_read, size_t num_extra_terms)
 {
+    UNUSED(num_extra_terms);
```

Comment:
Most of the times there are 3 situations that lead to unused parameter:
1. there is no other option. e.g. when the function has to comply with a certain signature
2. there is a bug
3. the parameter is completely useless
Using `UNUSED` works just for 1. In this case we are either in 2 or 3.

@fadushin said that we should remove the parameter, since it's likely 3.

So let's fix this change on another commit.